atlasdb-client/src/main/java/com/palantir/atlasdb/logging/LoggingArgs.java

```diff
@@ -64,6 +64,10 @@ static synchronized void setLogArbitrator(KeyValueServiceLogArbitrator arbitrato
         return getArg(argName, tableReference, logArbitrator.isTableReferenceSafe(tableReference));
     }
 
+    public static Arg<String> customTableName(TableReference tableReference, String tableName) {
+        return getArg("tableName", tableName, logArbitrator.isTableReferenceSafe(tableReference));
```

Comment:
Ah, okay. So this logs a `tableName` of the string provided, if the ref is safe.

Reply:
yep, exactly. kinda unfortunate that we need this but it was the easiest way i could see to support injecting the tablename in the format the cql requires